### PR TITLE
Update the demo scheme to be more generic

### PR DESCRIPTION
**Pull request by rafaelbernard**
Update the demo scheme to be more generic and include any other attributes we might want to use
https://app.asana.com/0/1171124854073033/1198870095630609

### CODE REVIEW

File: inz-demo/personal-data/Issues/ageLimits/description.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <Name>
+        <en>Demo Personal Data Age Limits</en>
+        <nl>Demo Personal Data Age Limits</nl>
+    </Name>
+    <ShortName>
+        <en>Person name</en>
+        <nl>Name</nl>
+    </ShortName>
+    <SchemeManager>inz-demo</SchemeManager>
+    <IssuerID>personal-data</IssuerID>
```
Comment (rafaelbernard) [inline inz-demo/personal-data/Issues/ageLimits/description.xml:11]: @camfindlay I changed `internal-affairs` to `personal-data`.

File: inz-demo/personal-data/Issues/ageLimits/description.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <CredentialID>ageLimits</CredentialID>
```
Comment (rafaelbernard) [inline inz-demo/personal-data/Issues/ageLimits/description.xml:12]: This is the `ageLimit` credential

**Review by ehsan-fj — APPROVED** [review]

File: inz-demo/description.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <SchemeManager version="7">
 	<Id>inz-demo</Id>
-	<Url>https://demo.irma.nz/inz-demo/</Url>
+	<Url>https://irma-scheme.local.internetnz.nz</Url>
```
Comment (ehsan-fj) [inline inz-demo/description.xml]: any reason to change this to local address?

Comment (rafaelbernard) [reply inz-demo/description.xml]: I am testing on my local and it was committed while I am changing things. I will leave this open to not forget to revert and sign again before the merge.

**Review by camfindlay — APPROVED** [review]
thanks the extra attributes are all good thanks.

**Pull request merged**